lib/jsdom/living/helpers/colors.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+const RGB_LOOKUP = {
+  black: [0x00, 0x00, 0x00],
+  silver: [0xC0, 0xC0, 0xC0],
+  gray: [0x80, 0x80, 0x80],
+  white: [0xFF, 0xFF, 0xFF],
+  maroon: [0x80, 0x00, 0x00],
+  red: [0xFF, 0x00, 0x00],
+  purple: [0x80, 0x00, 0x80],
+  fuchsia: [0xFF, 0x00, 0xFF],
+  green: [0x00, 0x80, 0x00],
+  lime: [0x00, 0xFF, 0x00],
+  olive: [0x80, 0x80, 0x00],
+  yellow: [0xFF, 0xFF, 0x00],
+  navy: [0x00, 0x00, 0x80],
+  blue: [0x00, 0x00, 0xFF],
+  teal: [0x00, 0x80, 0x80],
+  aqua: [0x00, 0xFF, 0xFF]
+};
+
+function colorKeywordToRGB(name) {
+  if (name.toLowerCase() in RGB_LOOKUP) {
```

Comment:
This will give weird results if someone passes `name = "constructor"` or similar. You can use `Object.hasOwn(RGB_LOOKUP, name.toLowerCase())`.